doc/nrf/known_issues.rst

```diff
@@ -1872,6 +1872,13 @@ NCSDK-17883: Cannot build peripheral UART sample with security (:kconfig:option:
 Bluetooth mesh
 ==============
 
+.. rst-class:: v2-4-0
+
+NCSDK-21590: Sensor client sample does not compile for nRF52832
+  Adding mesh shell support for the sensor client increase the need of RAM, and the sensor client sample is not able to compile because of shortage of RAM.
```

Comment:
```suggestion
  Adding mesh shell support for the sensor client sample increases the need for RAM.
  The sensor client sample is not able to compile because of RAM shortage.
```